packages/client-config/src/client-config-schema/client_config_v1.ts

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable */
```

Comment:
Could we generate this file during build and gitignore it?

Reply:
Reasons we shouldn't
1. Avoids adding a dependency in the library as it's not really needed.
2. If we are not 100% happy with the codegen, we need to override some types after the generation.
3. If we don't pin the codegen package, possibility of types changing. 

I don't see much value in moving this to the build process right now. If we do need in the future, we can always move it later.